exercises/practice/queen-attack/.meta/example.f90

```diff
@@ -0,0 +1,23 @@
+
+module queen_attack
+  implicit none
+contains
+
+  logical function isValid(pos)
+    integer, dimension(2) :: pos
+    ! check if valid
+    isValid = all(pos(:) > 0) .and. all(pos(:) < 9)
```

Comment:
Nice solution with array operations!

I think the comment may be a little redundant - the function name is `isValid`